qupath-core/src/test/java/qupath/lib/objects/classes/TestPathClassFactory.java

```diff
@@ -0,0 +1,143 @@
+/*-
+ * #%L
+ * This file is part of QuPath.
+ * %%
+ * Copyright (C) 2014 - 2016 The Queen's University of Belfast, Northern Ireland
+ * Contact: IP Management (ipmanagement@qub.ac.uk)
+ * Copyright (C) 2018 - 2021 QuPath developers, The University of Edinburgh
+ * %%
+ * QuPath is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * QuPath is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License 
+ * along with QuPath.  If not, see <https://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package qupath.lib.objects.classes;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.awt.Color;
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import qupath.lib.common.ColorTools;
+
+@SuppressWarnings("javadoc")
+public class TestPathClassFactory {
+	
+	@Test
+	public void test_getPathClass1() {
+		sameClass(new PathClass("Ignore*", ColorTools.makeRGB(180, 180, 180)), PathClassFactory.getPathClass(PathClassFactory.StandardPathClasses.IGNORE));
+		sameClass(new PathClass("Image", ColorTools.makeRGB(180, 180, 180)), PathClassFactory.getPathClass(PathClassFactory.StandardPathClasses.IMAGE_ROOT));
```

Comment:
This works for me in eclipse, but fails when running with `gradlew` with an `java.lang.UnsupportedOperationException` - which it should, if the class already exists.

I presume the order in which the tests run may be making a difference here.

(`makeRGB` should also now be `packRGB`)